lib/model/reading_time.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:easy_localization/easy_localization.dart';
```

Comment:
Why not just use built in DateTime object?

Reply:
The built-in package returns the whole time, either in days or hrs. If I set the reading time as 1 day 2 hrs 10 min, then we would be showing it as either 1 day or 26hrs. It'll not be precise if we show only in days, and hard to read if we show in hrs for longer durations.

Reply:
I see, I think this could be done as an extension to DateTime but I'm not sure exactly how. Just one ask - can it return "day" instead of "days" if the value is 1? Same for hours and minutes